### DISCUSSION TITLE
Enable Authentication and Authorization

### DISCUSSION THREAD

**Discussion by punchh-rakeshj** [provectus/kafka-ui#1997] [Q&A] [answered]
Is there any way to have basic authentication and authorization in kafka-ui?
any plan for ldap or SAML with Okta ?
any documentation for having google authentication?

**Comment by Haarolean** [comment, marked as answer]
Hey, thanks for reaching out.

Basic auth: [kafka-ui-auth-context.yaml](https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui-auth-context.yaml#L19), lines 19-21
Authorization: #753 (WIP)
LDAP: [auth-ldap.yaml](https://github.com/provectus/kafka-ui/blob/20986343e3c05da177070ac28a042951f8e4b861/docker/auth-ldap.yaml#L29), line 29
SAML: #478 (planned)
SSO (google does work, but note #1441): [SSO.md](https://github.com/provectus/kafka-ui/blob/master/documentation/guides/SSO.md)
